### PR TITLE
bees: Added a crude implementation of a better beeswax recipe

### PR DESCRIPTION
**Pull request by Sunconure11**
headdesk

### CODE REVIEW

File: src/java/growthcraft/bees/init/GrcBeesRecipes.java [hunk 1]
```diff
@@ -145,6 +147,11 @@ public void register()
 					honeyBottleStack, honeyBottleStack, honeyBottleStack,
 					Items.bucket);
 		}
+		
+	{
+		GrowthCraftCellar.boozeBuilderFactory.create(beeWaste.fluid.getFluid())
+			.brewsFrom(FluidRegistry.WATER(250), new ItemStack(GrowthCraftBees.ItemHoneyCombEmpty), TickUtils.minutes(1), new Residue(GrowthCraftBees.ItemBeesWax.asStack(1), 1.0f));
```
Comment (IceDragon200) [inline src/java/growthcraft/bees/init/GrcBeesRecipes.java]:
``` java
.brewsFrom(new FluidStack(FluidRegistry.WATER, 250), 
  GrowthCraftBees.items.honeyCombEmpty.asStack(), 
  TickUtils.minutes(1), 
  new Residue(GrowthCraftBees.items.beesWax.asStack(1), 1.0f));
```

File: src/java/growthcraft/bees/init/GrcBeesItems.java [hunk 2]
```diff
@@ -36,6 +36,7 @@
 import net.minecraft.item.ItemStack;
 import net.minecraftforge.oredict.OreDictionary;
 import net.minecraftforge.oredict.ShapelessOreRecipe;
+import net.minecraftforge.fluids.FluidRegistry;
```
Comment (IceDragon200) [inline src/java/growthcraft/bees/init/GrcBeesItems.java]: Wrong location m8

File: src/java/growthcraft/bees/init/GrcBeesRecipes.java [hunk 3]
```diff
@@ -43,6 +43,8 @@
 import net.minecraftforge.fluids.Fluid;
 import net.minecraftforge.fluids.FluidContainerRegistry;
 import net.minecraftforge.fluids.FluidStack;
+import growthcraft.api.core.GrcFluid;
```
Comment (IceDragon200) [inline src/java/growthcraft/bees/init/GrcBeesRecipes.java]: Move these up to the growthcraft imports, also added the FluidRegistry import here

File: src/java/growthcraft/bees/init/GrcBeesRecipes.java [hunk 4]
```diff
@@ -145,6 +148,14 @@ public void register()
 					honeyBottleStack, honeyBottleStack, honeyBottleStack,
 					Items.bucket);
 		}
+		
+	{
+		GrowthCraftCellar.boozeBuilderFactory.create(beeWaste.fluid.getFluid())
```
Comment (IceDragon200) [inline src/java/growthcraft/bees/init/GrcBeesRecipes.java]:
``` java
GrowthCraftCellar.boozeBuilderFactory.create(GrowthCraftBees.fluids.beeWaste.fluid.getFluid())
```

**Comment by Sunconure11** [comment]
FINALLY

**Comment by IceDragon200** [comment]
:tada:

**Pull request merged**